ui/src/routes/organizations/$orgId/products/$productId/repositories/$repoId/_repo-layout/create-run/-create-run-utils.ts

```diff
@@ -592,12 +622,85 @@ export function formValuesToPayload(
       }
     : undefined;
 
+  //
+  // Reporter configuration
+  //
+
+  // Check if CycloneDX, SPDX, and/or NOTICE file reports are enabled in the form,
+  // and configure them to use all output formats, accordingly.
```

Comment:
Here you say "all formats", which I like better than "both formats" in the commit message. Mind aligning the latter to this?